lib/src/git.rs

```diff
@@ -826,6 +833,13 @@ pub fn export_some_refs(
     mut_repo: &mut MutableRepo,
     git_ref_filter: impl Fn(GitRefKind, RemoteRefSymbol<'_>) -> bool,
 ) -> Result<Vec<FailedRefExport>, GitExportError> {
+    fn get<'a, V>(map: &'a [(RefName, V)], key: &RefName) -> Option<&'a V> {
```

Comment:
I'd change the argument name here to `sorted_map`. It's obvious from the code, but editor code hints when mousing over invocations of this functions won't show that.

I'm starting to wonder if we should have a `SortedVec` wrapper type that could inidcate this in type names. (I guess creating it would do the `sort_unstable_by`; I haven't fully thought it through, and perhaps there's already a crate available).

Reply:
I can rename it, but I think the debug assertion is good enough to describe the precondition.

Regarding `Sorted` wrapper, it might make sense to use something like that at public API boundary. I sometimes want to express that this `impl Iterator` is sorted, but that's not easy.